Add hendriknielaender/zvm to Package and Version Manager (#220)
zigcc/awesome-zig: README.md

- [tristanisham/zvm](https://github.com/tristanisham/zvm) - Lets you easily install/upgrade between different versions of Zig. ZLS install can be included. (written in Go).
- [rosarp/nu-zigup](https://github.com/rosarp/nu-zigup) - Download & manage Zig compilers & zls binaries. Script is written in nushell.
- [weezy20/zv](https://github.com/weezy20/zv) - Fast Zig/ZLS version manager + project starter kit written in Rust. Binaries available for macOS/Windows/Linux.

> - [hendriknielaender/zvm](https://github.com/hendriknielaender/zvm) - POSIX-compliant fast and simple Zig version manager.

Reviewer: I wonder what does `POSIX-compliant` mean in this context?

Author: To be honest, I just directly quoted the tagline from the repo's description. 😅
I guess the author meant its underlying scripts and shims strictly follow POSIX standards. But I agree it sounds a bit out of place in a simple list.
Should I just change it to something like: `A fast and simple Zig version manager`?

> - [hendriknielaender/zvm](https://github.com/hendriknielaender/zvm) - POSIX-compliant fast and simple Zig version manager.

Reviewer: ```suggestion
- [hendriknielaender/zvm](https://github.com/hendriknielaender/zvm) - A fast and simple Zig version manager written in Zig.
```
